packages/graphile-build-pg/src/plugins/PgColumnDeprecationPlugin.js

```diff
@@ -4,28 +4,19 @@ import type { Plugin } from "graphile-build";
 export default (function PgColumnDeprecationPlugin(builder) {
   builder.hook("GraphQLObjectType:fields:field", (field, build, context) => {
     const {
-      scope: {
-        isPgRowType,
-        isPgCompoundType,
-        pgIntrospection: table,
-        pgFieldIntrospection,
-      },
+      scope: { pgFieldIntrospection },
     } = context;
     if (
-      !(isPgRowType || isPgCompoundType) ||
-      !table ||
-      table.kind !== "class" ||
       !pgFieldIntrospection ||
-      !pgFieldIntrospection.tags
+      !pgFieldIntrospection.tags ||
+      !pgFieldIntrospection.tags.deprecated
     ) {
       return field;
     }
-    return pgFieldIntrospection.tags.deprecated
-      ? Object.assign({}, field, {
-          deprecationReason: Array.isArray(pgFieldIntrospection.tags.deprecated)
-            ? pgFieldIntrospection.tags.deprecated.join("\n")
-            : pgFieldIntrospection.tags.deprecated,
-        })
-      : field;
+    return Object.assign({}, field, {
```

Comment:
I prefer `{...field, `; "object spread properties" is supported as of Node 8.6 (which is the minimum requirement for PostGraphile for a reason 😉 ) according to https://node.green/